blog/2020-01-angular-scully.md

```diff
@@ -0,0 +1,332 @@
+---
+title: Create powerfull fast pre-rendered Angular Apps using _Scully_ static site generator
+description: 'With this blog post, I will show you how you can easily create an Angular blogging app by using the power of _Scully_ static site genrator. It will automatically detect all app routes and create static sites out of it that are ready to ship for production.'
```

Comment:
`genrator` should be `generator`. 

But, instead of saying the `power of Scully SSG`, you should say `the power of Angular, and then help of Scully`. At the end of the day, it's just Angular with a build process provided by @HeroDevs.